api/src/main/java/com/community/api/entity/Qualification.java

```diff
@@ -17,11 +17,17 @@
 public class Qualification
 {
     @Id
-    private Long qualification_id;
+    private Integer qualification_id;
 
     @Column(name = "qualification_name", nullable = false)
     private String qualification_name;
 
     @Column(name = "qualification_description", nullable = false)
     private String qualification_description;
+
+    @Column(name = "is_subjects_required")
+    private Boolean is_subjects_required;
+
+    @Column(name = "is_stream_required")
+    private Boolean is_stream_required;
```

Comment:
To avoid null default values that can cause rollback errors, you can modify your column definition like this:

@Column(name = "is_subjects_required", nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
private Boolean isSubjectsRequired;
This ensures that the column defaults to false instead of null, preventing rollback errors caused by null values in the database.